src/actions/reformat.py

```diff
@@ -0,0 +1,19 @@
+from talon import Module, actions
+from .get_texts import get_texts
+
+
+mod = Module()
+
+
+@mod.action_class
+class Actions:
+    def cursorless_reformat(targets: dict, formatters: str):
+        """Reformat targets with formatter"""
+        texts = get_texts(targets, show_decorations=False)
+        updated_texts = list(map(
+            lambda text: actions.user.reformat_text(text, formatters),
```

Comment:
this one also needs to be in knausj?

Reply:
Yes read the description of the pr. This one is already listed there. The voice command for this one is commented away in the talon file